templates/pg_hba.template

```diff
@@ -0,0 +1,3 @@
+local   all     all     trust
```

Comment:
If I understand correctly, then `trust` means: no password needed, which might potentially expose other databases running on the system.

What about replacing `trust` with `md5`?

Reply:
Ok, I see it's not that bad
https://www.postgresql.org/docs/9.1/auth-methods.html#AUTH-TRUST

Still, if I remember correctly, then during the tutorial in Oslo, things started working on CentOS when we specified `md5` (not 100% sure)

Reply:
With `md5` we need to change how `postgres` user operates. E.g. we would have to define a password. That is not done on the Ubuntu setup I believe. And I suppose the setup is in fact in `trust` by default, but I have not checked. Just did this to be closest to the existing setup. Oslo worked when we configured `trust`, but also `md5` after configuring a password for `postgres`.

Reply:
Ok, then let's go with `trust` for the moment